src/metaTypes.h

```diff
@@ -35,18 +36,18 @@ namespace METAIO_NAMESPACE {
 typedef char                MET_ASCII_CHAR_TYPE;
 typedef char                MET_CHAR_TYPE;
 typedef unsigned char       MET_UCHAR_TYPE;
-typedef short               MET_SHORT_TYPE;
-typedef unsigned short      MET_USHORT_TYPE;
-typedef int                 MET_INT_TYPE;
-typedef unsigned int        MET_UINT_TYPE;
-typedef long                MET_LONG_TYPE;
-typedef unsigned long       MET_ULONG_TYPE;
+typedef int16_t             MET_SHORT_TYPE;
+typedef uint16_t            MET_USHORT_TYPE;
+typedef int32_t             MET_INT_TYPE;
+typedef uint32_t            MET_UINT_TYPE;
+typedef int32_t             MET_LONG_TYPE;
+typedef uint32_t            MET_ULONG_TYPE;
 #if defined(_WIN32) && !defined(__CYGWIN__) && !defined(__MING_W32__)
```

Comment:
Not sure, but since I mostly develop on Linux, I didn't want to break something that was working. A different commit could remove this. This would be cleaner as this might potentially create some issues with very old compilers.